src/GeositeFramework/css/app-print.css

```diff
@@ -95,4 +119,16 @@
 
 .instructions {
     display: none;
-}
\ No newline at end of file
+}
+
+.legend-close {
+    visibility: hidden;
+}
+
+.dojoResizeHandle {
+    visibility: hidden !important;
+}
+
+#export-title {
```

Comment:
Good catch, I missed that one. I'm guessing #767 meant centered vertically and laterally, is that correct @mmcfarland ?